src/main.ts

```diff
@@ -9,6 +9,8 @@ async function run(): Promise<void> {
     const apply = core.getBooleanInput('apply')
 
     const image = dockerfile.load(dockerfile_path)
+    await image.init_package_manager()
```

Comment:
Omitting this call would lead to a unusable image object.
I'd rather write:
```suggestion
    const image = await dockerfile.load(dockerfile_path)
```
and have the initialization dealt with at instantiation.